src/factory.ts

```diff
@@ -131,7 +123,7 @@ export async function buildStrategy(
     github: options.github,
     changelogSections: options.changelogSections,
   });
-  const strategyOptions: BaseStrategyOptions = {
+  const strategyOptions: AnyStrategyOptions = {
```

Comment:
I'm thinking about changing `buildStrategy` to use expansion instead of assign of each field manually.

Sidenote: There is case difference in one option:
`ReleaserConfig.skipGithubRelease`
vs
`BaseStrategyOptions.skipGitHubRelease`
rest is 1:1.

```ts
    const strategyOptions: BaseStrategyOptions = {
    ...options,
    skipGitHubRelease: options.skipGithubRelease,
    targetBranch,
    versioningStrategy,
    changelogNotes,
  };
```

It is not needed, just an idea.

Reply:
that sounds find to me - can you add a comment to the `skipGitHubRelease: options.skitGithubRelease` so that we can rename/clean it up in the next major version.